### ISSUE TITLE
env setting doesn't seem to be respected

### ISSUE THREAD

**Issue by jamonholmgren** [google/closure-compiler-js#88]
TL;DR version: `compile({ env: 'CUSTOM' })` doesn't seem to set the `env` to `CUSTOM` (it remains `BROWSER`).

## Background

I'm trying to use `google-closure-compiler-js` with Rollup. I initially used [rollup-plugin-closure-compiler-js](https://github.com/camelaissani/rollup-plugin-closure-compiler-js).

```js
    closure({
      createSourceMap: true,
      env: 'CUSTOM',
      processCommonJsModules: true
    }),
```

It would give me this error:

```
Invalid module path "path" for resolution mode "BROWSER"
```

I figured it was a problem with the Rollup plugin, so I ditched that and made my own inline plugin, utilizing this npm package directly. You can see my code here:

https://github.com/infinitered/gluegun/commit/0cfb2315be8f1b65cdb66db86a645f71031ffeeb

When I try to run Rollup, I still get an error:

```
...
{ file: 'Input_0',
  description: 'Invalid module path "app-module-path" for resolution mode "BROWSER"',
  type: 'JSC_INVALID_MODULE_PATH',
  lineNo: 50,
  charNo: 36 }
[!] (closure-compiler-js plugin) Error: Error transforming bundle with 'closure-compiler-js' plugin: compilation errors: 44
```

## Reproduce Steps

If you want, you can clone the repo and repro the problem like so (assumes you have `yarn` installed):

```
git clone git@github.com:infinitered/gluegun.git
cd gluegun
git checkout google-closure-compiler
yarn install
yarn build
```

You should see the same output as me, shown above.

## As a global install

This _works_ if you install globally and run it manually, setting the `--env` to CUSTOM. First, follow the steps above, but comment out the `closure` call in [rollup.config.js](https://github.com/infinitered/gluegun/blob/google-closure-compiler/rollup.config.js#L43-L47) before running `yarn build`. Then, run this:

```
npm i -g google-closure-compiler-js
google-closure-compiler-js ./dist/index.js --env=CUSTOM > test.js
```

You'll see that it compiles just fine.

My hypothesis is that `env` isn't being set. Unfortunately, I don't know how `gcc-js` is created well enough to know where to trace the problem. Thank you for your help and let me know if I can help in any way.

**Comment by mathiasrw** [comment]
Any news on this?

**Comment by ChadKillingsworth** [comment]
Sorry for the delay. The `env` flag is definitely already supported.

From reading the error message is that module resolution requires a valid source path - and it must start with a `./`, `../` or `/`. That's what the error message is telling you.

Documentation for how to pass proper paths to the compiler is at https://github.com/google/closure-compiler-npm#javascript-version-1

I recommend switching to the https://github.com/google/closure-compiler-npm repo. All future development is based there.
